website/source/docs/secrets/jwt/index.html.md

```diff
@@ -10,21 +18,21 @@ The JWT secret backend acts like the `transit` backend, it does not store any in
 
 ## Algorithms
 
-### RSA 
+#### RSA
```

Comment:
The header size was IMHO to big: ![Green is the new one](https://i.imgur.com/ckaIOLH.png)